doc/source/contributing.rst

```diff
@@ -333,10 +341,10 @@ Requirements
 ~~~~~~~~~~~~
 
 To build the *pandas* docs there are some extra requirements: you will need to
-have ``sphinx`` and ``ipython`` installed. `numpydoc
+have sphinx and ipython installed. `numpydoc
```

Comment:
I am personally in favor here of keeping some mark-up (so they stand out a bit from the text so it is clearer those packages are needed). But I don't think we are very consistent on this throughout the docs (also not for pandas itself, for which you seem to use italics here in this PR?)